itou/templates/job_seekers_views/job_applications.html

```diff
@@ -0,0 +1,81 @@
+{% extends "layout/base.html" %}
+{% load matomo %}
+{% load static %}
+{% load str_filters %}
+{% load format_filters %}
+{% load url_add_query %}
+
+{% block title %}
+    Profil de {{ job_seeker.get_full_name|mask_unless:can_view_personal_information }} {{ block.super }}
+{% endblock %}
+
+{% block title_content %}
+    <div class="d-flex flex-column flex-md-row gap-3 mb-3 justify-content-md-between">
+        <h1 class="mb-0">Candidat : {{ job_seeker.get_full_name|mask_unless:can_view_personal_information }}</h1>
+        {% url 'search:employers_results' as search_url %}
+        {% if can_view_personal_information %}
+            {% url_add_query search_url job_seeker=job_seeker.public_id city=job_seeker.city_slug as url_query %}
+        {% else %}
+            {% url_add_query search_url job_seeker=job_seeker.public_id as url_query %}
+        {% endif %}
+        <a href="{{ url_query }}" {% matomo_event "candidature" "clic" "postuler-pour-ce-candidat" %} class="btn btn-lg btn-primary btn-ico">
+            <i class="ri-draft-line fw-medium" aria-hidden="true"></i>
+            <span>Postuler pour ce candidat</span>
+        </a>
+    </div>
+{% endblock %}
+
+{% block title_extra %}
+    <ul class="s-tabs-01__nav nav nav-tabs mb-0" role="tablist" data-it-sliding-tabs="true">
+        <li class="nav-item" role="presentation">
+            <a class="nav-link"
+               href="{% url 'job_seekers_views:details' public_id=job_seeker.public_id %}?back_url={% url 'job_seekers_views:list' %}"
+               aria-controls="informations"
+               aria-selected="true"
+               {% matomo_event "candidat" "clic-onglet" "informations candidat" %}>Informations générales</a>
+        </li>
+        <li class="nav-item" role="presentation">
+            <a class="nav-link active"
+               href="{% url 'job_seekers_views:job_applications' public_id=job_seeker.public_id %}?back_url={% url 'job_seekers_views:list' %}"
+               aria-controls="candidatures"
+               aria-selected="false"
```

Comment:
```suggestion
               aria-selected="false"
               {% matomo_event "candidat" "clic-onglet" "informations candidat" %}>Informations générales</a>
        </li>
        <li class="nav-item" role="presentation">
            <a class="nav-link active"
               href="{% url 'job_seekers_views:job_applications' public_id=job_seeker.public_id %}?back_url={% url 'job_seekers_views:list' %}"
               aria-controls="candidatures"
               aria-selected="true"
```

Je me demande si ça ne vaudrait pas le coup d'utiliser un partiel pour rendre ce block et que l'on include avec le tab actif en contexte